tests/TenantDatabaseManagerTest.php

```diff
@@ -99,20 +99,21 @@
     Event::listen(TenancyEnded::class, RevertToCentralContext::class);
 
     $tenant = Tenant::create();
+    $tenantHostConnectionName = config('tenancy.database.tenant_host_connection_name');
```

Comment:
You don't need the variable, you can hardcode the string. Just like `central` is hardcoded.